src/libs/actions/BankAccounts.js

```diff
@@ -340,9 +340,7 @@ function fetchFreePlanVerifiedBankAccount(stepToOpen) {
     // Remember which account BankAccountStep subStep the user had before so we can set it later
     const subStep = lodashGet(reimbursementAccountInSetup, 'subStep', '');
 
-    // We are using set here since we will rely on data from the server (not local data) to populate the VBA flow
-    // and determine which step to navigate to.
-    Onyx.set(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {loading: true, error: ''});
+    Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {loading: true, error: ''});
```

Comment:
Kinda want to propose a less risky change here. Here's how it goes...

Since the `/bank-account` flow may be expecting multiple fields to be "reset" before we fetch the bank account then we should try to only carry over the state that we need _for now_. 

I think to solve the current issue we could do something like:

```js
const initialData = {loading: true, error: ''};
const state = lodashGet(reimbursementAccountInSetup, 'achData.state');

if (state === BankAccount.STATE.OPEN) {
    initialData.achData = {state: BankAccount.STATE.OPEN};
}

Onyx.set(ONYXKEYS.REIMBURSEMENT_ACCOUNT, initialData);
```

Reply:
I've pushed changes for solution #1 (with the flag). LMK what you think